webpack/assets/javascripts/react_app/components/PF4/InlineEdit/InlineEdit.stories.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import Story from '../../../../../../stories/components/Story';
+import InlineEdit from './InlineEdit';
+
+export default {
+  title: 'Components|InlineEdit',
+};
+
+// eslint complains about hooks in storybooks
+/* eslint-disable react-hooks/rules-of-hooks */
+export const defaultStory = () => {
+  const [value, setValue] = useState('hello');
+
+  const onEdit = async (v, a) => {
+    // delay to emulate an asynchronous backend call
+    await (async () => new Promise(resolve => setTimeout(resolve, 1000)))();
+    setValue(v);
+    alert(`Update of ${a} successful`);
+  };
+  return (
+    <Story>
+      <InlineEdit
+        value={value}
+        attribute="backend_identifier"
+        onEdit={onEdit}
+      />
+    </Story>
+  );
+};
+
+export const withTextArea = () => {
```

Comment:
I think this and the next story could be consolidated using Knobs (see `EmptyState.stories.js` for examples).  You can add a `boolean` for `textArea` and a `text` for `value`, and the user could just play around with it in Storybook.

Reply:
good suggestion, I updated to use knobs, but I did keep both stories since I like the verbose-ness of having the separate stories since knobs is more of a power-user type feature and might not be apparent to everyone. The knobs are cool though! (I've never used it before)